### PR TITLE
Enable XPI signoffs

### PR DESCRIPTION
**Pull request by rail**
* Fix permission checking in `phase_signoff()`
* Remove old relman users
* Some UI clean up (sorry)

### CODE REVIEW

File: api/src/shipit_api/admin/api.py [hunk 1]
```diff
@@ -279,6 +279,12 @@ def phase_signoff(name, phase, body):
     if who in [s.completed_by for s in phase_obj.signoffs]:
         abort(409, f"Already signed off by {who}")
 
+    # signoff.permissions corresponds to the group in settings.py
+    groups = current_app.config.get("GROUPS", {})
+    group = groups.get(signoff.permissions, [])
+    if who not in group:
+        abort(401, f"User `{who} is not in the `{signoff.permissions}` group")
```
Comment (escapewindow) [inline api/src/shipit_api/admin/api.py:286]: Nit: we may want to add a closing backtick after `{who}`.

File: api/src/shipit_api/admin/settings.py [hunk 2]
```diff
@@ -72,8 +72,8 @@
         "sfraser@mozilla.com",
         "tprince@mozilla.com",
     ],
-    "firefox-signoff": ["rkothari@mozilla.com", "ehenry@mozilla.com", "jcristau@mozilla.com", "pchevrel@mozilla.com", "rvandermeulen@mozilla.com"],
-    "fenix-signoff": ["rkothari@mozilla.com", "ehenry@mozilla.com", "jcristau@mozilla.com", "pchevrel@mozilla.com", "rvandermeulen@mozilla.com"],
+    "firefox-signoff": ["jcristau@mozilla.com", "pchevrel@mozilla.com", "rvandermeulen@mozilla.com"],
```
Comment (escapewindow) [inline api/src/shipit_api/admin/settings.py:75]: Will this break anything around automatic betas? Does relman know they will need to sign off for Firefox releases?

File: api/src/shipit_api/admin/api.py (hunk 1 above)
Comment (rail) [reply api/src/shipit_api/admin/api.py:286]: bah, missed that

File: api/src/shipit_api/admin/settings.py (hunk 2 above)
Comment (rail) [reply api/src/shipit_api/admin/settings.py:75]: This PR enables signoffs only for the XPI promote phase. Everything else is unaffected.

**Pull request merged**